css/signuppages.css

```diff
@@ -0,0 +1,314 @@
+body {
+	padding: 0;
+	margin: 0;
+	background: #ffffff;
+}
+
+/* @extend display-flex; */
+display-flex, .display-flex, .display-flex-center {
+    display: flex;
+    display: -webkit-flex; }
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/BuildForSDGCohort2/team-040-frontend/pullRequest?prid=6093393)